Npgsql.EntityFramework/NpgsqlProviderManifest.cs

```diff
@@ -10,16 +10,27 @@
 #endif
 using System.Xml;
 using System.Data;
+using NpgsqlTypes;
 
 namespace Npgsql
 {
     internal class NpgsqlProviderManifest : DbXmlEnabledProviderManifest
     {
+
+        static NativeToBackendTypeConverterOptions _redshiftConverterOptions = new NativeToBackendTypeConverterOptions(false, false, false, null);
+
         public NpgsqlProviderManifest(string serverVersion)
             : base(CreateXmlReaderForResource("Npgsql.NpgsqlProviderManifest.Manifest.xml"))
         {
+
+            if (serverVersion == "8.0.2")
+                ConverterOptions = _redshiftConverterOptions;
```

Comment:
@franciscojunior, is this necessary? We already have a property "SupportsEStringPrefix" on the connector, which is itself based on the server version. Should this be automatically consulted, rather than a hard-coded condition here with a redshift-specific conversion options?

What I mean, is that it would be better to fix E-syntax for all old databases, rather than making specific cases for redshift...